plugins/catalog-backend-module-ldap/src/ldap/client.ts

```diff
@@ -147,9 +147,10 @@ export class LdapClient {
   async searchStreaming(
     dn: string,
     options: SearchOptions,
-    f: (entry: SearchEntry) => void,
```

Comment:
maybe this should be `void | Promise<void>` in order to keep backwards compatibility?